src/github.com/getlantern/flashlight/pac.go

```diff
@@ -80,9 +80,19 @@ func genPACFile() {
 	formatter :=
 		`var bypassDomains = %s;
 		function FindProxyForURL(url, host) {
-			if (host == "localhost" || host == "127.0.0.1") {
+			if (isPlainHostName(host) // including localhost
```

Comment:
Oh I see -- they're primarily referring to dots as in "internet.com" not as in "1.1.1.1" -- looks good.